From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Milan Burda <milan.burda@gmail.com>
Date: Sat, 28 Sep 2019 11:01:51 +0200
Subject: fix: when building with enable_plugins=false

Fix building with enable_plugins=false. This patch should be upstreamed.

diff --git a/content/browser/sandbox_parameters_mac.mm b/content/browser/sandbox_parameters_mac.mm
index 5eead918eb9d9df03c86b5201b3f924643707f4b..e1e07fc8cecbf26dc80f62c222b14f7a0be1b0c0 100644
--- a/content/browser/sandbox_parameters_mac.mm
+++ b/content/browser/sandbox_parameters_mac.mm
@@ -25,12 +25,16 @@
 #include "content/public/common/content_client.h"
 #include "content/public/common/content_features.h"
 #include "content/public/common/content_switches.h"
-#include "content/public/common/pepper_plugin_info.h"
+#include "ppapi/buildflags/buildflags.h"
 #include "sandbox/mac/seatbelt_exec.h"
 #include "services/service_manager/sandbox/mac/sandbox_mac.h"
 #include "services/service_manager/sandbox/sandbox_type.h"
 #include "services/service_manager/sandbox/switches.h"
 
+#if BUILDFLAG(ENABLE_PLUGINS)
+#include "content/public/common/pepper_plugin_info.h"
+#endif
+
 namespace content {
 
 namespace {
@@ -152,7 +156,9 @@ void SetupPPAPISandboxParameters(sandbox::SeatbeltExecClient* client) {
   SetupCommonSandboxParameters(client);
 
   std::vector<content::WebPluginInfo> plugins;
+#if BUILDFLAG(ENABLE_PLUGINS)
   PluginService::GetInstance()->GetInternalPlugins(&plugins);
+#endif
 
   base::FilePath bundle_path = service_manager::SandboxMac::GetCanonicalPath(
       base::mac::MainBundlePath());
